api/app/GraphQL/Queries/CountPoolCandidatesByPool.php

```diff
@@ -35,7 +35,7 @@ public function __invoke($_, array $args)
         PoolCandidate::scopeAvailable($queryBuilder);
 
         // expiry status filter (filter active pool candidates)
-        PoolCandidate::scopeExpiryFilter($queryBuilder, ['expiryStatus' => ApiEnums::CANDIDATE_EXPIRY_FILTER_ACTIVE]);
+        PoolCandidate::scopeExpiryStatus($queryBuilder, ApiEnums::CANDIDATE_EXPIRY_FILTER_ACTIVE);
```

Comment:
Form of the other expiry scope is the same as the suspended scope, not an array of one key/value pair. 

![image](https://github.com/GCTC-NTGC/gc-digital-talent/assets/40485260/044a97e1-54a9-4fa5-a9e0-29c4162216bb)